### DISCUSSION TITLE
[Icon request] Bars center (center aligned text)

### DISCUSSION THREAD

**Discussion by knd775** [tailwindlabs/heroicons#1150] [Ideas]
I am trying to recreate the image below with heroicons, but there is nothing for the second one. This is pretty important for creating a text editor UI.

![image](https://github.com/tailwindlabs/heroicons/assets/1235055/2448f53a-7883-42c8-8f01-0799562dae73)

**Comment by ndri** [comment]
I added `bars-3-bottom-center` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=bars-3-bottom-center
